ntbs-service/Models/Notification.cs

```diff
@@ -80,12 +80,34 @@ public class Notification : ModelBase
         public int? TotalContactsStartedTreatment => CalculateSum(ContactTracing.AdultsStartedTreatment, ContactTracing.ChildrenStartedTreatment);
         public int? TotalContactsFinishedTreatment => CalculateSum(ContactTracing.AdultsFinishedTreatment, ContactTracing.ChildrenFinishedTreatment);
 
+        public string PatientEditPath => GetNotificationEditPath("Patient");
```

Comment:
Decided to define properties for these rather than writing out the whole path in the view